### ISSUE TITLE
Only ELG required, disable-x11 build broken. 

### ISSUE THREAD

**Issue by joshuafc** [NVIDIA/libglvnd#209]
ubuntu running inside nvidia-docker2 container.

libglvnd-1.2.0.tar.gz from release page.

configure with 

```
./configure --prefix=/usr --disable-x11 --disable-glx --disable-gles --enable-x11=no
```

while make,  errors:
```
make[4]: Entering directory '/tmp/libglvnd-1.2.0/src/EGL'
  CC       libEGL_la-libegl.lo
In file included from ../../include/EGL/egl.h:39:0,
                 from ../../include/glvnd/libeglabi.h:34,
                 from libeglabipriv.h:39,
                 from libegl.c:44:
../../include/EGL/eglplatform.h:122:22: fatal error: X11/Xlib.h: No such file or directory
compilation terminated.
```

**Comment by kbrenneman** [comment]
Libglvnd is hosted on Freedesktop now, so any bug reports need to go there:
https://gitlab.freedesktop.org/glvnd/libglvnd

**Comment by kbrenneman** [comment]
Also, this is already fixed in 1.3.0:
https://gitlab.freedesktop.org/glvnd/libglvnd/-/merge_requests/193
